src/elements/content-sidebar/activity-feed/activity-feed/ActivityThread.scss

```diff
@@ -36,3 +52,50 @@
         outline: auto;
     }
 }
+
+.bcs-ActivityItem.bcs-is-focused .bcs-ActivityThread {
+    padding-bottom: 0;
+    box-shadow: none;
+
+    &,
+    &:hover {
+        background-color: $white;
+    }
+
+    .bcs-ActivityThread-selectWrapper {
+        margin: 0 $sidebarActivityFeedSelectedItemSpacingHorizontal;
+        padding-bottom: $sidebarActivityFeedSpacingVertical - $sidebarActivityFeedSelectedItemBorderWidth;
```

Comment:
We could probably simplify a lot of the padding calculations we're doing by using a `box-shadow` that's styled to look like a border.